charts/sonarqube/static_compatibility_test/default-values.yaml

```diff
@@ -0,0 +1 @@
+#default value file
```

Comment:
As discussed, we could activate all the resources in default template:
- ServiceAccount and attributes
- Change admin password (.Values.account.adminPassword)
- Persistence
- Network Policies
- Prometheus?
- monitoringPasscode
- and activate the attributes...